index_v1.html

```diff
@@ -1,42 +1,101 @@
+<?php
+$currentPage = 'login_1';
+include 'nav-login.php';
+
+$error = '';
+if (isset($_GET['error'])) {
+	$error = htmlspecialchars($_GET['error']); // Use htmlspecialchars to avoid XSS attacks
+}
+?>
 <!DOCTYPE html>
-<html lang="en">

```

Comment:
I deleted this part of teh code, as it didn't have any use and was just laying around as extra code